### ISSUE TITLE
`docz dev` compile failed on 0.9.3. dependencies were not found: @babel/runtime

### ISSUE THREAD

**Issue by jigsawye** [pedronauck/docz#190]
## Bug Report

**Describe the bug**

After upgrade docz to 0.9.3, the `docz dev` is failed, `docz build` also failed too.

**To Reproduce**

1. Upgrade docz from 0.8.0 to 0.9.3
2. run `docz dev`
3. See error

```shell
ERROR  Failed to compile with 8 errors                                        15:11:16

These dependencies were not found:

* @babel/runtime/helpers/builtin/es6/assertThisInitialized in ./node_modules/react-powerplug/dist/react-powerplug.esm.js
* @babel/runtime/helpers/builtin/es6/defineProperty in ./node_modules/react-powerplug/dist/react-powerplug.esm.js
* @babel/runtime/helpers/builtin/es6/extends in ./node_modules/react-spring/dist/web.js
* @babel/runtime/helpers/builtin/es6/inheritsLoose in ./node_modules/react-powerplug/dist/react-powerplug.esm.js, ./node_modules/react-spring/dist/web.js
* @babel/runtime/helpers/builtin/es6/objectSpread in ./node_modules/react-powerplug/dist/react-powerplug.esm.js
* @babel/runtime/helpers/builtin/es6/objectWithoutProperties in ./node_modules/react-powerplug/dist/react-powerplug.esm.js, ./node_modules/react-spring/dist/web.js

To install them, you can run: npm install --save @babel/runtime/helpers/builtin/es6/assertThisInitialized @babel/runtime/helpers/builtin/es6/defineProperty @babel/runtime/helpers/builtin/es6/extends @babel/runtime/helpers/builtin/es6/inheritsLoose @babel/runtime/helpers/builtin/es6/objectSpread @babel/runtime/helpers/builtin/es6/objectWithoutProperties
```

**Enviroment**

* OS: OSX 10.13.6
* Node: 10.7.0
* Yarn: 1.9.2

**Additional context/Screenshots**

`docz dev` with `--debug`

```
ℹ ｢serve｣: Serving Static Content from: node_modules/docz-core/dist/non-existent/
ℹ ｢hot｣: WebSocket Server Listening on 127.0.0.1:60757
Happy[jsx]: Version: 5.0.0. Threads: 3
ℹ ｢serve｣: Project is running at http://127.0.0.1:3030
ℹ ｢serve｣: Server URI copied to clipboard
Happy[jsx]: All set; signaling webpack to proceed.
Happy[mdx]: Version: 5.0.0. Threads: 3
Happy[mdx]: All set; signaling webpack to proceed.
ℹ ｢hot｣: Applying DefinePlugin:__hotClientOptions__
ℹ ｢hot｣: webpack: Compiling...
ℹ ｢hot｣: webpack: Compiling Done
✖ ｢wdm｣: Hash: 5cc1f948fef5762f44c2
Version: webpack 4.16.4
Time: 12577ms
Built at: 2018-08-04 15:17:29
                           Asset      Size               Chunks  Chunk Names
      static/js/message-index.js  27.1 KiB        message-index  message-index
        static/js/alert-index.js  12.5 KiB          alert-index  alert-index
       static/js/button-index.js  24.6 KiB         button-index  button-index
         static/js/card-index.js  25.9 KiB           card-index  card-index
     static/js/checkbox-index.js  36.6 KiB       checkbox-index  checkbox-index
     static/js/dropdown-index.js    23 KiB       dropdown-index  dropdown-index
         static/js/grid-index.js  28.9 KiB           grid-index  grid-index
         static/js/grid-space.js   9.2 KiB           grid-space  grid-space
      static/js/heading-index.js    26 KiB        heading-index  heading-index
         static/js/icon-index.js  25.1 KiB           icon-index  icon-index
              static/js/index.js  10.2 KiB                index  index
        static/js/input-index.js  32.6 KiB          input-index  input-index
         static/js/menu-index.js  23.2 KiB           menu-index  menu-index
                static/js/app.js    80 KiB                  app  app
        static/js/modal-index.js  9.94 KiB          modal-index  modal-index
        static/js/radio-index.js    36 KiB          radio-index  radio-index
        static/js/runtime~app.js    31 KiB          runtime~app  runtime~app
       static/js/select-index.js  19.9 KiB         select-index  select-index
         static/js/spin-index.js  10.3 KiB           spin-index  spin-index
       static/js/switch-index.js  12.8 KiB         switch-index  switch-index
        static/js/table-index.js    20 KiB          table-index  table-index
         static/js/tabs-index.js  45.5 KiB           tabs-index  tabs-index
   static/js/text-field-index.js  20.9 KiB     text-field-index  text-field-index
       static/js/theme-colors.js  16.5 KiB         theme-colors  theme-colors
static/js/theme-spacing-scale.js  43.8 KiB  theme-spacing-scale  theme-spacing-scale
      static/js/tooltip-index.js  27.7 KiB        tooltip-index  tooltip-index
            static/js/vendors.js  16.9 MiB              vendors  vendors
Entrypoint app = static/js/runtime~app.js static/js/vendors.js static/js/app.js
[./.docz/app/index.jsx] 1.85 KiB {app} [built]
[./.docz/app/root.jsx] 1.48 KiB {app} [built]
[./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js] 212 bytes {vendors} [built]
[./node_modules/react-dom/index.js] 1.33 KiB {vendors} [built]
[./node_modules/react/index.js] 190 bytes {vendors} [built]
[./node_modules/regenerator-runtime/runtime.js] 23.6 KiB {vendors} [built]
[./node_modules/webpack-hot-client/client/hot.js] (webpack)-hot-client/client/hot.js 4.91 KiB {vendors} [built]
[./node_modules/webpack-hot-client/client/index.js?46fbd0e1-62bb-4db0-9bb0-6e1f978d3f84] (webpack)-hot-client/client?46fbd0e1-62bb-4db0-9bb0-6e1f978d3f84 2.49 KiB {vendors} [built]
[./node_modules/webpack-hot-client/client/log.js] (webpack)-hot-client/client/log.js 2.82 KiB {vendors} [built]
[./node_modules/webpack-hot-client/client/socket.js] (webpack)-hot-client/client/socket.js 1.45 KiB {vendors} [built]
[./node_modules/webpack-serve-overlay/errorOverlay.js] (webpack)-serve-overlay/errorOverlay.js 4.46 KiB {vendors} [built]
[./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 509 bytes {vendors} [built]
[./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 519 bytes {vendors} [built]
[0] multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx 40 bytes {app} [built]
[1] multi webpack-hot-client/client?46fbd0e1-62bb-4db0-9bb0-6e1f978d3f84 ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx 52 bytes {app} [built]
    + 2467 hidden modules

ERROR in ./node_modules/react-powerplug/dist/react-powerplug.esm.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/assertThisInitialized' in '/Users/jigsawye/projects/ui/node_modules/react-powerplug/dist'
 @ ./node_modules/react-powerplug/dist/react-powerplug.esm.js 3:0-94 62:108-130 62:131-153 62:229-251 62:252-274 74:18-40 421:108-130 421:131-153 423:24-46 423:47-69 423:121-143 423:144-166 427:24-46 427:47-69 437:24-46 437:47-69 439:24-46 439:47-69 443:24-46 443:47-69 445:18-40
 @ ./node_modules/docz-theme-default/dist/index.m.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx

ERROR in ./node_modules/react-powerplug/dist/react-powerplug.esm.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/defineProperty' in '/Users/jigsawye/projects/ui/node_modules/react-powerplug/dist'
 @ ./node_modules/react-powerplug/dist/react-powerplug.esm.js 4:0-80 62:92-107 62:213-228 421:92-107 423:8-23 423:105-120 427:8-23 437:8-23 439:8-23 443:8-23
 @ ./node_modules/docz-theme-default/dist/index.m.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx

ERROR in ./node_modules/react-spring/dist/web.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/extends' in '/Users/jigsawye/projects/ui/node_modules/react-spring/dist'
 @ ./node_modules/react-spring/dist/web.js 1:0-66 513:32-40 565:53-61 900:11-19 918:43-51 919:11-19 946:36-44 1433:14-22 1554:46-54 1610:9-17 1687:34-42 1738:24-32 1756:36-44 1769:13-21 1840:34-42 1841:58-66 1882:39-47 1898:82-90 1944:9-17 1989:18-26 2019:19-27 2030:21-29 2076:16-24 2088:41-49 2170:16-24 2184:41-49 2234:22-30 2325:65-73 2326:44-52 2331:14-22 2359:44-52 2373:32-40 2375:9-17 2450:55-63 2464:51-59 2496:44-52 2498:13-21 2620:36-44 2661:13-21 2678:13-21 2702:9-17
 @ ./src/Alert/index.js
 @ ./src/index.js
 @ ./src/theme/colors.mdx
 @ ./.docz/app/imports.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx

ERROR in ./node_modules/react-powerplug/dist/react-powerplug.esm.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/inheritsLoose' in '/Users/jigsawye/projects/ui/node_modules/react-powerplug/dist'
 @ ./node_modules/react-powerplug/dist/react-powerplug.esm.js 2:0-78 53:2-16 412:2-16
 @ ./node_modules/docz-theme-default/dist/index.m.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx

ERROR in ./node_modules/react-spring/dist/web.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/inheritsLoose' in '/Users/jigsawye/projects/ui/node_modules/react-spring/dist'
 @ ./node_modules/react-spring/dist/web.js 2:0-78 558:2-16 613:2-16 654:2-16 735:2-16 1091:2-16 1243:2-16 1369:2-16 1425:2-16 1495:6-20 1616:2-16 1956:2-16 2124:2-16 2207:2-16 2409:2-16 2532:2-16
 @ ./src/Alert/index.js
 @ ./src/index.js
 @ ./src/theme/colors.mdx
 @ ./.docz/app/imports.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx

ERROR in ./node_modules/react-powerplug/dist/react-powerplug.esm.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/objectSpread' in '/Users/jigsawye/projects/ui/node_modules/react-powerplug/dist'
 @ ./node_modules/react-powerplug/dist/react-powerplug.esm.js 1:0-76 62:172-185 307:13-26 313:14-27 547:13-26 742:11-24
 @ ./node_modules/docz-theme-default/dist/index.m.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx

ERROR in ./node_modules/react-powerplug/dist/react-powerplug.esm.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/objectWithoutProperties' in '/Users/jigsawye/projects/ui/node_modules/react-powerplug/dist'
 @ ./node_modules/react-powerplug/dist/react-powerplug.esm.js 6:0-98 101:14-38 169:14-38 190:14-38 227:14-38 257:14-38 304:14-38 346:14-38 378:14-38 486:14-38 544:14-38 597:14-38 635:14-38 667:14-38 698:14-38
 @ ./node_modules/docz-theme-default/dist/index.m.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx

ERROR in ./node_modules/react-spring/dist/web.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/objectWithoutProperties' in '/Users/jigsawye/projects/ui/node_modules/react-spring/dist'
 @ ./node_modules/react-spring/dist/web.js 5:0-98 1025:21-45 1859:18-42 1928:13-37 1974:16-40 2040:14-38 2074:16-40 2086:17-41 2161:16-40 2321:15-39 2489:16-40
 @ ./src/Alert/index.js
 @ ./src/index.js
 @ ./src/theme/colors.mdx
 @ ./.docz/app/imports.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx
Child html-webpack-plugin for "index.html":
         Asset      Size  Chunks  Chunk Names
    index.html  1.38 MiB       0
    Entrypoint undefined = index.html
    [./node_modules/html-webpack-plugin/lib/loader.js!./.docz/app/index.html] 562 bytes {0} [built]
    [./node_modules/lodash/lodash.js] 527 KiB {0} [built]
    [./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 509 bytes {0} [built]
    [./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 519 bytes {0} [built]
ℹ ｢wdm｣: Failed to compile.
ℹ ｢hot｣: webpack: Bundle Invalidated
ℹ ｢wdm｣: Compiling...
ℹ ｢hot｣: webpack: Compiling...
ℹ ｢hot｣: webpack: Compiling Done
✖ ｢wdm｣: Hash: 1a9a73a353cc42b74e84
Version: webpack 4.16.4
Time: 240ms
Built at: 2018-08-04 15:17:30
                               Asset      Size               Chunks  Chunk Names
          static/js/message-index.js  27.1 KiB        message-index  message-index
cf8b2aecd1fe252daebd.hot-update.json  44 bytes
                    static/js/app.js    80 KiB                  app  app
           static/js/button-index.js  24.6 KiB         button-index  button-index
             static/js/card-index.js  25.9 KiB           card-index  card-index
         static/js/checkbox-index.js  36.6 KiB       checkbox-index  checkbox-index
         static/js/dropdown-index.js    23 KiB       dropdown-index  dropdown-index
             static/js/grid-index.js  28.9 KiB           grid-index  grid-index
             static/js/grid-space.js   9.2 KiB           grid-space  grid-space
          static/js/heading-index.js    26 KiB        heading-index  heading-index
             static/js/icon-index.js  25.1 KiB           icon-index  icon-index
                  static/js/index.js  10.2 KiB                index  index
            static/js/input-index.js  32.6 KiB          input-index  input-index
             static/js/menu-index.js  23.2 KiB           menu-index  menu-index
            static/js/alert-index.js  12.5 KiB          alert-index  alert-index
            static/js/modal-index.js  9.94 KiB          modal-index  modal-index
            static/js/radio-index.js    36 KiB          radio-index  radio-index
            static/js/runtime~app.js    31 KiB          runtime~app  runtime~app
           static/js/select-index.js  19.9 KiB         select-index  select-index
             static/js/spin-index.js  10.3 KiB           spin-index  spin-index
           static/js/switch-index.js  12.8 KiB         switch-index  switch-index
            static/js/table-index.js    20 KiB          table-index  table-index
             static/js/tabs-index.js  45.5 KiB           tabs-index  tabs-index
       static/js/text-field-index.js  20.9 KiB     text-field-index  text-field-index
           static/js/theme-colors.js  16.5 KiB         theme-colors  theme-colors
    static/js/theme-spacing-scale.js  43.8 KiB  theme-spacing-scale  theme-spacing-scale
          static/js/tooltip-index.js  27.7 KiB        tooltip-index  tooltip-index
                static/js/vendors.js  16.9 MiB              vendors  vendors
Entrypoint app = static/js/runtime~app.js static/js/vendors.js static/js/app.js
[./.docz/app/index.jsx] 1.85 KiB {app}
[./.docz/app/root.jsx] 1.48 KiB {app}
[./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js] 212 bytes {vendors}
[./node_modules/react-dom/index.js] 1.33 KiB {vendors}
[./node_modules/react/index.js] 190 bytes {vendors}
[./node_modules/regenerator-runtime/runtime.js] 23.6 KiB {vendors}
[./node_modules/webpack-hot-client/client/hot.js] (webpack)-hot-client/client/hot.js 4.91 KiB {vendors}
[./node_modules/webpack-hot-client/client/index.js?46fbd0e1-62bb-4db0-9bb0-6e1f978d3f84] (webpack)-hot-client/client?46fbd0e1-62bb-4db0-9bb0-6e1f978d3f84 2.49 KiB {vendors}
[./node_modules/webpack-hot-client/client/log.js] (webpack)-hot-client/client/log.js 2.82 KiB {vendors}
[./node_modules/webpack-hot-client/client/socket.js] (webpack)-hot-client/client/socket.js 1.45 KiB {vendors}
[./node_modules/webpack-serve-overlay/errorOverlay.js] (webpack)-serve-overlay/errorOverlay.js 4.46 KiB {vendors}
[./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 509 bytes {vendors}
[./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 519 bytes {vendors}
[0] multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx 40 bytes {app}
[1] multi webpack-hot-client/client?46fbd0e1-62bb-4db0-9bb0-6e1f978d3f84 ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx 52 bytes {app}
    + 2467 hidden modules

ERROR in ./node_modules/react-powerplug/dist/react-powerplug.esm.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/assertThisInitialized' in '/Users/jigsawye/projects/ui/node_modules/react-powerplug/dist'
 @ ./node_modules/react-powerplug/dist/react-powerplug.esm.js 3:0-94 62:108-130 62:131-153 62:229-251 62:252-274 74:18-40 421:108-130 421:131-153 423:24-46 423:47-69 423:121-143 423:144-166 427:24-46 427:47-69 437:24-46 437:47-69 439:24-46 439:47-69 443:24-46 443:47-69 445:18-40
 @ ./node_modules/docz-theme-default/dist/index.m.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx

ERROR in ./node_modules/react-powerplug/dist/react-powerplug.esm.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/defineProperty' in '/Users/jigsawye/projects/ui/node_modules/react-powerplug/dist'
 @ ./node_modules/react-powerplug/dist/react-powerplug.esm.js 4:0-80 62:92-107 62:213-228 421:92-107 423:8-23 423:105-120 427:8-23 437:8-23 439:8-23 443:8-23
 @ ./node_modules/docz-theme-default/dist/index.m.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx

ERROR in ./node_modules/react-spring/dist/web.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/extends' in '/Users/jigsawye/projects/ui/node_modules/react-spring/dist'
 @ ./node_modules/react-spring/dist/web.js 1:0-66 513:32-40 565:53-61 900:11-19 918:43-51 919:11-19 946:36-44 1433:14-22 1554:46-54 1610:9-17 1687:34-42 1738:24-32 1756:36-44 1769:13-21 1840:34-42 1841:58-66 1882:39-47 1898:82-90 1944:9-17 1989:18-26 2019:19-27 2030:21-29 2076:16-24 2088:41-49 2170:16-24 2184:41-49 2234:22-30 2325:65-73 2326:44-52 2331:14-22 2359:44-52 2373:32-40 2375:9-17 2450:55-63 2464:51-59 2496:44-52 2498:13-21 2620:36-44 2661:13-21 2678:13-21 2702:9-17
 @ ./src/Alert/index.js
 @ ./src/Alert/index.mdx
 @ ./.docz/app/imports.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx

ERROR in ./node_modules/react-powerplug/dist/react-powerplug.esm.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/inheritsLoose' in '/Users/jigsawye/projects/ui/node_modules/react-powerplug/dist'
 @ ./node_modules/react-powerplug/dist/react-powerplug.esm.js 2:0-78 53:2-16 412:2-16
 @ ./node_modules/docz-theme-default/dist/index.m.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx

ERROR in ./node_modules/react-spring/dist/web.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/inheritsLoose' in '/Users/jigsawye/projects/ui/node_modules/react-spring/dist'
 @ ./node_modules/react-spring/dist/web.js 2:0-78 558:2-16 613:2-16 654:2-16 735:2-16 1091:2-16 1243:2-16 1369:2-16 1425:2-16 1495:6-20 1616:2-16 1956:2-16 2124:2-16 2207:2-16 2409:2-16 2532:2-16
 @ ./src/Alert/index.js
 @ ./src/Alert/index.mdx
 @ ./.docz/app/imports.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx

ERROR in ./node_modules/react-powerplug/dist/react-powerplug.esm.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/objectSpread' in '/Users/jigsawye/projects/ui/node_modules/react-powerplug/dist'
 @ ./node_modules/react-powerplug/dist/react-powerplug.esm.js 1:0-76 62:172-185 307:13-26 313:14-27 547:13-26 742:11-24
 @ ./node_modules/docz-theme-default/dist/index.m.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx

ERROR in ./node_modules/react-powerplug/dist/react-powerplug.esm.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/objectWithoutProperties' in '/Users/jigsawye/projects/ui/node_modules/react-powerplug/dist'
 @ ./node_modules/react-powerplug/dist/react-powerplug.esm.js 6:0-98 101:14-38 169:14-38 190:14-38 227:14-38 257:14-38 304:14-38 346:14-38 378:14-38 486:14-38 544:14-38 597:14-38 635:14-38 667:14-38 698:14-38
 @ ./node_modules/docz-theme-default/dist/index.m.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx

ERROR in ./node_modules/react-spring/dist/web.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/objectWithoutProperties' in '/Users/jigsawye/projects/ui/node_modules/react-spring/dist'
 @ ./node_modules/react-spring/dist/web.js 5:0-98 1025:21-45 1859:18-42 1928:13-37 1974:16-40 2040:14-38 2074:16-40 2086:17-41 2161:16-40 2321:15-39 2489:16-40
 @ ./src/Alert/index.js
 @ ./src/Alert/index.mdx
 @ ./.docz/app/imports.js
 @ ./.docz/app/root.jsx
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/docz-core/node_modules/babel-polyfill/lib/index.js ./.docz/app/index.jsx
Child html-webpack-plugin for "index.html":
                                   Asset      Size  Chunks  Chunk Names
                              index.html  1.38 MiB       1
    cf8b2aecd1fe252daebd.hot-update.json  44 bytes
    Entrypoint undefined = index.html
    [./node_modules/html-webpack-plugin/lib/loader.js!./.docz/app/index.html] 562 bytes {1} [built]
    [./node_modules/lodash/lodash.js] 527 KiB {1}
    [./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 509 bytes {1}
    [./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 519 bytes {1}
ℹ ｢wdm｣: Failed to compile.
```

**Comment by pedronauck** [comment]
Can you give a little shot of your `package.json` and your `babelrc`?

**Comment by clauderic** [comment]
Same issue here. Here's what my `package.json` file looks like:
```json
  "scripts": {
    "docz:dev": "docz dev",
    "docz:build": "docz build"
  },
  "dependencies": {
    "prop-types": "^15.5.7"
  },
  "peerDependencies": {
    "react": "15.x || 16.x"
  },
  "devDependencies": {
    "@babel/cli": "^7.0.0-beta.56",
    "@babel/core": "^7.0.0-beta.56",
    "@babel/preset-env": "^7.0.0-beta.56",
    "@babel/preset-react": "^7.0.0-beta.56",
    "@babel/runtime": "^7.0.0-beta.56",
    "@types/jest": "^20.0.8",
    "@types/prop-types": "^15.5.1",
    "@types/react": "^15.0.38",
    "@types/react-dom": "^15.5.4",
    "awesome-typescript-loader": "^3.2.1",
    "babel-eslint": "7.0.0",
    "babel-jest": "^23.4.2",
    "codecov": "^1.0.1",
    "docz": "^0.9.3",
    "docz-theme-default": "^0.9.3",
    "eslint": "^4.10.0",
    "eslint-plugin-shopify": "^22.1.0",
    "fs-extra": "^4.0.1",
    "jest": "^19.0.2",
    "nwb": "0.15.x",
    "react": "^15.4.2",
    "react-addons-test-utils": "^15.4.2",
    "react-dom": "^15.4.2",
    "react-test-renderer": "^15.4.2",
    "rimraf": "^2.6.1",
    "rollup": "^0.49.2",
    "rollup-plugin-babel": "^3.0.2",
    "rollup-plugin-commonjs": "^8.2.0",
    "rollup-plugin-node-resolve": "^3.0.0",
    "rollup-plugin-typescript2": "^0.15.0",
    "rollup-plugin-uglify": "^2.0.1",
    "tslint": "^5.10.0",
    "tslint-config-shopify": "^3.0.2",
    "typescript": "^3.0.1"
  },
```

Don't have a `.babelrc` file, should I add one? If so, what should the presets be? Didn't see it mentioned in the docs

**Comment by jigsawye** [comment]
@pedronauck Here is my `package.json` and `.babelrc`:

`package.json`

```
 "dependencies": {
    "polished": "^1.9.3",
    "prop-types": "^15.6.2",
    "ramda": "^0.25.0",
    "react": "^16.4.2",
    "react-dom": "^16.4.2",
    "react-icons": "^2.2.7",
    "react-powerplug": "^1.0.0-rc.1",
    "react-select": "^2.0.0",
    "react-spring": "^5.5.4",
    "react-textarea-autosize": "^7.0.4",
    "styled-components": "^3.4.0",
    "styled-normalize": "^8.0.0",
    "styled-system": "^3.0.2",
    "styled-tools": "^1.0.0",
    "stylelint-config-prettier": "^3.3.0"
  },
  "devDependencies": {
    "@babel/cli": "^7.0.0-beta.56",
    "@babel/core": "^7.0.0-beta.56",
    "@babel/plugin-proposal-class-properties": "^7.0.0-beta.56",
    "@babel/preset-env": "^7.0.0-beta.56",
    "@babel/preset-react": "^7.0.0-beta.56",
    "babel-core": "^7.0.0-0",
    "babel-eslint": "^8.2.6",
    "babel-jest": "^23.4.2",
    "babel-plugin-ramda": "^1.6.2",
    "babel-plugin-styled-components": "^1.5.1",
    "cross-env": "^5.2.0",
    "docz": "^0.8.0",
    "enzyme": "^3.3.0",
    "enzyme-adapter-react-16": "^1.1.1",
    "enzyme-to-json": "^3.3.4",
    "eslint": "^4.19.1",
    "eslint-config-yoctol": "^0.18.0",
    "eslint-import-resolver-jest": "^2.1.1",
    "eslint-plugin-import": "^2.13.0",
    "eslint-plugin-jsx-a11y": "^6.1.1",
    "eslint-plugin-prettier": "^2.6.2",
    "eslint-plugin-react": "^7.10.0",
    "eslint-plugin-sort-imports-es6-autofix": "^0.3.0",
    "husky": "^0.14.3",
    "jest": "^23.4.2",
    "jest-styled-components": "^5.0.1",
    "lint-staged": "^7.2.0",
    "prettier": "^1.14.0",
    "prettier-package-json": "^1.6.0",
    "stylelint": "9.4.0",
    "stylelint-config-styled-components": "^0.1.1",
    "stylelint-config-yoctol": "^1.1.1",
    "stylelint-processor-styled-components": "^1.3.2"
  },
```

`.babelrc`
```
{
  "presets": ["@babel/preset-env", "@babel/preset-react"],
  "plugins": [
    "babel-plugin-styled-components",
    "@babel/plugin-proposal-class-properties",
    "ramda"
  ]
}
```

**Comment by pedronauck** [comment]
Fixed on release [v0.9.5](https://github.com/pedronauck/docz/releases/tag/v0.9.5)

**Comment by clauderic** [comment]
Thanks for the quick response @pedronauck :heart:

**Comment by pedronauck** [comment]
It was a dependency bundled as external that was causing that bug, sorry guys 🙏 Sometimes are very hard to catch some errors 😕 

Your welcome @clauderic 🙌🏻

**Comment by clauderic** [comment]
I have the same error using `0.9.5` actually 😢

**Comment by jigsawye** [comment]
Looks the issue still happens in my project 😭 

@clauderic can you explain how to resolve it ?

**Comment by pedronauck** [comment]
Fixed on [v0.9.6](https://github.com/pedronauck/docz/releases/tag/v0.9.6) @jigsawye ✌️
